src/coreclr/tools/aot/ILCompiler.LLVM/Compiler/RyuJitLLVMCompilation.cs

```diff
@@ -0,0 +1,200 @@
+// // Licensed to the .NET Foundation under one or more agreements.
+// // The .NET Foundation licenses this file to you under the MIT license.
+//
```

Comment:
Delete? The whole file is commented out.

Reply:
Done.